awx/ui/docs/JobOutput.md

```diff
@@ -2,15 +2,39 @@ This document is meant to provide some guidance into the functionality of Job Ou
 
 ## Overview of the feature/screen. Summary of what it does/is
 
-1. Elapsed time / unfollow button
-2. Page up and page down buttons
-3. Unique qualities of the different job types.
+Joboutput is a feature that allows users to see how their job is doing as it is being run.
+This feature displays data sent to the UI via websockets that are connected to several different end
+points in the API.
 
-- Some don’t allow search by event data and thus Event is not an option in the drop down
-- Some don’t have expand, collapse
+Generally speaking the job output has 2 different states that result in different functionality.
+The first state the output could be in is when the job is actively running. During this time there is limited
+functionality because of how the job events are processed when they reach the UI. While the job is running, and
+output is coming into the UI we turn off the following features:
 
-4. Differences in the output from when a job is running and when a job is complete.
-5. Which features are enabled when it’s running and which aren’t.
+1. [Search](#Search)- The ability to search the output of a job.
+2. [Expand/Collapse](#Expand/Collapse)- The ability to expand and collapse job events, tasks, plays, or even the
+   job itself. The only part of the job ouput that is not collapsable is the playbook summary (Only jobs that
+   are executed from a Job Template have Expand/Collapse functionality).
+
+We turn on the following features:
+
+1. Follow/unfollow - If you are `Following` this means that you are streaming the output ont the screen
+   as it comes into the UI. If you see some output that you want to examine closer, while the job is running
+   scroll to it, and click unfollow, and the output will stop streaming onto the screen. This feature is only
+   enabled when the job is running and is not complete.
+2. Page up and page down buttons- These buttons allow you to navigate quickly up and down the output.
+   The first two buttons scroll up or down.
+   ![Running job](images/JobOutput-running.png)
```

Comment:
It would be more helpful to the reader if the image was annotated (with arrows or a box around) pointing to the buttons you are describing here.